### DISCUSSION TITLE
Github contributions dissappears

### DISCUSSION THREAD

**Discussion by aglago** [community/community#136238] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My Github contributions from last year have all disappeared, and I think that was caused by removing one of my emails from my github account. Is there anyway I can get my contributions back?? Even after adding the email I don't see my contributions coming back!

**Comment by davevad93** [comment, marked as answer]
Hi @aglago ,

The [docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#your-local-git-commit-email-isnt-connected-to-your-account) state the following:

- ***If the email address used for the commit is not connected to your account on GitHub, you must [add the email address](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account) to your account on GitHub. Your contributions graph will be rebuilt automatically when you add the new address.***

Technically by re-associating and verifying the email used for your commits to your GitHub account your contribution graph will be updated accordingly; try waiting 24 hours because the contribution graph may take a while to update.

To avoid this problem in the future in case you change your email address, I advise you to select the email privacy feature and use the [noreply address](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address#about-commit-email-addresses) provided by GitHub for all your commits.

**Reply by aglago** [reply]
thank you dave, hope to see some good news tomorrow:)

**Reply by davevad93** [reply]
You're welcome @aglago , hope it gets fixed! 👍

**Reply by ykylejan** [reply]
did this work? its been more than 48 hours, but my contributions graph did not go back from its former state after re-linking & verifying my email again..

**Reply by aglago** [reply]
it worked for me

**Reply by nuexq** [reply]
its been days for me and the graph didn't update, but the commits I made with the old email does read as mine

**Reply by PRANAV-SHILU** [reply]
i have similar problem, i relinked previous email but still my previous contribution is not showing till after 3 dats, kindly help

**Comment by Pasanlaksitha** [comment]
response #38693
